services/feeds.py

```
@@ -137,6 +167,52 @@ def generate_sitemap(self, blogs_data: List[Dict[str, Any]]) -> str:
+            # Try frontmatter date
+            if frontmatter.get('date'):
+                return frontmatter['date']
+            
+            # Use blog date
+            if date_str:
+                return date_str
+            
+            # Fallback to current date
+            return datetime.utcnow().strftime('%Y-%m-%d')
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)